requirements/base.txt

```diff
@@ -24,6 +24,7 @@ pillow==6.2.2
 pypandoc==1.4
 python-dateutil==2.8.1
 pytz==2019.3
+django-rosetta==0.9.3
```

Comment:
I don't think rosetta really belongs in our app since it can't be used in our production workflows. Translations for version-controlled messages need to happen in translatewiki.